pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023. SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.demo.ui.player
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsDraggedAsState
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.media3.common.Player
+import ch.srgssr.pillarbox.demo.ui.player.controls.PlayerControls
+import ch.srgssr.pillarbox.demo.ui.player.controls.PlayerError
+import ch.srgssr.pillarbox.demo.ui.player.controls.PlayerNoContent
+import ch.srgssr.pillarbox.ui.ScaleMode
+import ch.srgssr.pillarbox.ui.exoplayer.ExoPlayerSubtitleView
+import ch.srgssr.pillarbox.ui.extension.hasMediaItemsAsState
+import ch.srgssr.pillarbox.ui.extension.playbackStateAsState
+import ch.srgssr.pillarbox.ui.extension.playerErrorAsState
+import ch.srgssr.pillarbox.ui.widget.ToggleableBox
+import ch.srgssr.pillarbox.ui.widget.keepScreenOn
+import ch.srgssr.pillarbox.ui.widget.player.PlayerSurface
+import ch.srgssr.pillarbox.ui.widget.rememberDelayedVisibilityState
+
+/**
+ * Simple player view
+ *
+ * @param player The [Player] to observe.
+ * @param modifier The modifier to be applied to the layout.
+ * @param scaleMode The surface scale mode.
+ * @param controlsVisible The control visibility.
+ * @param controlsToggleable The controls are toggleable.
+ * @param bottomActionsContent The action to display under the slider.
+ */
+@Composable
+fun PlayerView(
+    player: Player,
+    modifier: Modifier = Modifier,
+    scaleMode: ScaleMode = ScaleMode.Fit,
+    controlsVisible: Boolean = true,
+    controlsToggleable: Boolean = true,
+    bottomActionsContent: @Composable ColumnScope.() -> Unit = {},
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>ComposableLambdaParameterNaming: Composable lambda parameter should be named `content`